File: OO-design/Calculator/java/Calculator.java
class Calculator {
    private int firstOperand;
    private int secondOperand;

    public Calculator(int firstOperand, int secondOperand) {
        this.firstOperand = firstOperand;
        this.secondOperand = secondOperand;
    }

    public int getFirstOperand() {
        return firstOperand;
    }

    public int getSecondOperand() {
        return secondOperand;
    }

    public void setFirstOperand(int firstOperand) {
        this.firstOperand = firstOperand;
    }

    public void setSecondOperand(int secondOperand) {
        this.secondOperand = secondOperand;
    }

    public int getAdditionResult() {
        return this.firstOperand + this.secondOperand;
    }

    public int getSubtractionResult() {
        return this.firstOperand - this.secondOperand;
    }

    public int getMultiplicationResult() {
        return this.firstOperand * this.secondOperand;
    }

    public int getDivisionResult() {
        return this.firstOperand / this.secondOperand;
    }
}

class Main {
    public static void main(String[] args) {
        Calculator calculator = new Calculator(3, 2);
        System.out.println(calculator.getAdditionResult());
        System.out.println(calculator.getSubtractionResult());

        calculator.setFirstOperand(6);
        calculator.setSecondOperand(3);
        System.out.println(calculator.getMultiplicationResult());
        System.out.println(calculator.getDivisionResult());
    }
}

File: OO-design/Calculator/php/Calculator.php
<?php

class Calculator {
    private int $firstOperand;
    private int $secondOperand;

    public function __construct(int $firstOperand, int $secondOperand) {
        $this->firstOperand = $firstOperand;
        $this->secondOperand = $secondOperand;
    }

    public function getFirstOperand(): int {
        return $this->firstOperand;
    }

    public function getSecondOperand(): int {
        return $this->secondOperand;
    }

    public function setFirstOperand(int $firstOperand): void {
        $this->firstOperand = $firstOperand;
    }

    public function setSecondOperand(int $secondOperand): void {
        $this->secondOperand = $secondOperand;
    }

    public function add(): int {
        return $this->firstOperand + $this->secondOperand;
    }

    public function subtract(): int {
        return $this->firstOperand - $this->secondOperand;
    }

    public function multiply(): int {
        return $this->firstOperand * $this->secondOperand;
    }

    public function divide(): float {
        return $this->firstOperand / $this->secondOperand;
    }
}

$calculator = new Calculator(3, 2);
echo $calculator->add() . PHP_EOL;
echo $calculator->subtract() . PHP_EOL;

$calculator->setFirstOperand(6);
$calculator->setSecondOperand(3);
echo $calculator->multiply() . PHP_EOL;
echo $calculator->divide() . PHP_EOL;